man/man8/zpool-upgrade.8

```diff
@@ -31,9 +30,10 @@
 .Dd August 9, 2019
 .Dt ZPOOL-UPGRADE 8
 .Os
+.
 .Sh NAME
 .Nm zpool-upgrade
-.Nd Manage version and feature flags of ZFS storage pools
+.Nd manage version and feature flags of ZFS storage pools
```

Comment:
maybe: "manage the version and feature flags of ZFS storage pools"

Reply:
Don't see it. There aren't specific flags here

Reply:
I was suggesting using the definite article "the" because there is only one version in a pool.

Reply:
"version" is a property of a "ZFS storage pool", not an object within it